### ISSUE TITLE
`Press Escape key to blur from terminal` shows when terminal does not have focus

### ISSUE THREAD

**Issue by richard-cox** [rancher/dashboard#13555]
**Setup**
- Rancher version: v2.11.0 (dev of)

**Describe the bug**
- The `Press Escape key to blur from terminal` shows even when focus is outside of the windows manager
- Given that the user can then use the escape key (and other keyboard shortcuts) normally without pressing esc, having a message asking to press esc to do so is misleading
- Does this apply to other use cases of the `Press x to blur` concept?

**To Reproduce**
- cluster explorer --> open kube shell (cursor in terminal flashes) --> click outside of the windows manager that contains the terminal

**Result**
- `Press Escape key to blur from terminal` is shown

**Expected Result**
- `Press Escape key to blur from terminal` is not show

**Comment by IsaSih** [comment]
Feature works as expected. 

Validated in 
Rancher | v2.11-a2ff78325d1768b0a52fa65a88c00a1f3cddace5-head
Dashboard | master 8e2d825
Rancher CLI | v2.11.0-rc.1
